### PR TITLE
Use Terminal lexer for console examples

### PR DESCRIPTION
**Pull request by wouterj**
This allows to fix https://github.com/symfony/symfony-docs/issues/6865

### CODE REVIEW

**Comment by javiereguiluz** [comment]
This seems to work ... but it introduces some issues: comments are now considered prompts. For example, in this page: http://pr-6930-6qmocelev2lwe.eu.platform.sh/setup.html  

highlight-error

I'm afraid it's "the curse of Sphinx": when someone tries to fix something ... they end up breaking more things 😭

**Comment by stof** [comment]
yeah, this is because a shell session does not have comments AFAIK

**Comment by wouterj** [comment]
status: needs work

We can quite easily fix this by creating a custom lexer that will call the bash lexer for lines starting with `$`/`C:\>` and treats lines starting with `#` as a comment. Will work on this, as it should not be that hard.

**Comment by wouterj** [comment]
Wow, love carsonbot :)

@javiereguiluz I've updated the PR with a very simple custom lexer. As far as I can see, it works. It uses 2 new styles not yet defined by symfony.com: `Generic.Prompt` (`$` and `C:\>`) and `Generic.Output` (command output).

Let me know if you like this change.

File: _build/conf.py [hunk 1]
```diff
@@ -108,6 +109,7 @@
 lexers['rst'] = RstLexer()
 lexers['varnish3'] = CLexer()
 lexers['varnish4'] = CLexer()
+lexers['terminal'] = TerminalLexer()
```
Comment (wouterj) [inline _build/conf.py]: @javiereguiluz you need to add this to the `conf.py` used by the doc builder. You probably also have to add `sys.path.append(os.path.abspath('_theme/_exts'))` (where `os.path.abspath('_theme/_exts')` should be replaced by the path to the `_theme/_exts` directory)

**Comment by javiereguiluz** [comment]
@wouterj we've updated symfony.com's Sphinx builder to include your TerminalLexer. Changes will be deployed soon and after that, it will be safe to merge this PR and regenerate the docs.

**Comment by javiereguiluz** [comment]
The changes have been implemented on symfony.com, so we can merge this PR and see how the new docs are generated. Thanks.

**Comment by xabbuh** [comment]
Okay, let's see how this looks like. :)

**Pull request closed without merging**